app/views/incomeSources/add/IncomeSourceReportingMethod.scala.html

```diff
@@ -113,7 +114,7 @@
         <li id="bullet-3">@Html(messages("incomeSources.add.incomeSourceReportingMethod.description4.bullet3"))</li>
     </ul>
 
-    @h1(messages("incomeSources.add.incomeSourceReportingMethod.chooseReport"), classes = "govuk-heading-m")
+    @h2(messages("incomeSources.add.incomeSourceReportingMethod.chooseReport"))
```

Comment:
any reason why we are changing H1 -> H2 ?

Reply:
h1 should only be for the main heading this is a subheading